server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/GetVacationResponseTest.java

```diff
@@ -72,12 +79,112 @@ public void teardown() {
     }
 
     @Test
-    public void getVacationResponseIsNotImplementedYet() {
+    public void getVacationResponseShouldReturnDefaultValue() {
+        given()
+            .accept(ContentType.JSON)
+            .contentType(ContentType.JSON)
+            .header("Authorization", accessToken.serialize())
+            .body("[[" +
+                "\"getVacationResponse\", " +
+                "{}, " +
+                "\"#0\"" +
+                "]]")
+        .when()
+            .post("/jmap")
+        .then()
+            .statusCode(200)
+            .body(NAME, equalTo("vacationResponse"))
+            .body(ARGUMENTS + ".accountId", equalTo(username))
```

Comment:
Sorry I didn't read the spec, but can you have a non null accountId in the response if you have a null account id in the request?